docs/DeveloperGuide.md

```diff
@@ -155,14 +155,14 @@ This section describes some noteworthy details on how certain features are imple
 
 `[written by: Yang Yue]`
 
-The mark done feature allows users to mark a specific appointment in the address book as done using `d/` tag and `t/` tag to specify the appointment at a certain time slot.
+The mark done feature allows users to mark a specific appointment in the appointment book as done using `APPT_INDEX` to specify the appointment in the list of appointments.
 
 #### 1.1 Implementation
-Command: `done d/DATE t/TIME`
+Command: `done APPT_INDEX`
 
 Example Usage:
-* `done d/Monday t/9am`
-* `done d/12-12-2020 t/12pm`
+* `done 1`
+* `done 4`
 
```

Comment:
Should also update line 173: `1. User types done d/DATE t/TIME into the app.`  
Should also remove line 169 (about `DateTimeLoader`)

Ps: Writing here bcos I cant comment on other lines.